README.md

```diff
@@ -36,15 +36,15 @@ $ sudo dnf install -y libusbx-devel systemd-devel
 ### 1. Set the Cargo runner
 
 The recommend way to use `probe-run` is to set as the Cargo runner of your application.
-Add this line to your Cargo configuration file (.cargo/config) and adjust ${PROBE_RUN_CHIP} for the particular `--chip` value:
+Add this line to your Cargo configuration file (.cargo/config) and set the particular `--chip` value
+for your target (in this case, the nRF52840):
 
 
 ``` toml
-[target.'cfg(all(target_arch = "arm", target_os = "none"))']
-runner = "probe-run --chip ${PROBE_RUN_CHIP}"
```

Comment:
`${PROBE_RUN_CHIP}` is not just string to replace. It is environment variable.

Reply:
Yes. However, I felt that the env var approach was covered by the next few paragraphs. Also, the way it is now confused me - I actually thought that it represented some cargo related variable.

Reply:
I'd agree to keep the generic example, but would also be fine with additionally having a specific one. What do you think about moving the generic one down to where the env var approach is mentioned?

Reply:
Sure. Can you please make a suggestion and then I'll commit it. Thanks.

Reply:
Sadly, I can't, since github only allows me to make suggestions for lines you touched and the ones closely above and below. I will write my suggestion here in a moment.

Reply:
### 1. Set the Cargo runner

The recommend way to use `probe-run` is to set as the Cargo runner of your application.

Add these two lines to your Cargo configuration file (`.cargo/config.toml`) and set the particular `--chip` value for your target. In this case it is `nRF52840_xxAA` for the [nRF52840]:

``` toml
[target.'cfg(all(target_arch = "arm", target_os = "none"))']
runner = "probe-run --chip nRF52840_xxAA"
#                          ^^^^^^^^^^^^^
```

To list all supported chips run `probe-run --list-chips`.

#### **1.1 Env variable**

To support multiple devices, or permit overriding default behavior, you may prefer to:
1. set the `${PROBE_RUN_CHIP}` environment variable, and
2. set `runner` (or `CARGO_TARGET_${TARGET_ARCH}_RUNNER`) to `probe-run`:

``` toml
[target.'cfg(all(target_arch = "arm", target_os = "none"))']
runner = "probe-run"
```

#### **1.2 Multiple probes**

If you have several probes connected, you can specify which one to use by adding the `--probe` option to the `runner` or setting the `${PROBE_RUN_PROBE}` environment variable with a value containing either `${VID}:${PID}` or `${VID}:${PID}:${SERIAL}`:

```console
// --probe
$ probe-run --probe '0483:3748' --chip ${PROBE_RUN_CHIP}

// PROBE_RUN_PROBE
$ PROBE_RUN_PROBE='1366:0101:123456' cargo run
```

To list all connected probes, run `probe-run --list-probes`.

[nRF52840]: https://www.nordicsemi.com/Products/Low-power-short-range-wireless/nRF52840

Reply:
That's the whole section "1. Set the Cargo runner".

Reply:
I've deleted it here and added it as a suggestion for line 39, but is a change for the whole section.